about.html

```diff
@@ -4,9 +4,106 @@
+          <a href="" onclick="menuClose()">News</a>
+      </div>
+    </div>
+  <header class="primary-header">
+    <ul id="social-icons">
+      <li class="media-icons"><img src="./images/facebook.png" alt=""></li>
+      <li class="media-icons"><img src="./images/twitter.png" alt=""></li>
+      <li class="media-icons"><img src="./images/linkedin.png" alt=""></li>
+    </ul>
+  </header>
+    <header class="toolbar">
+      <img src="./images/summer.png" alt="" class="summer_img" >
+      <div >
+        <img src='./images/mobile_menu.png' alt="mobile_menu" id="toolbar-img" onclick= menuOpen() >
+      </div>
+      <ul id="link">
+        <li class="links"><a href="index.html">Home</a></li>
+        <li class="links"><a href="#">About</a></li>
+        <li class="links"><a href="#">Program</a></li>
+        <li class="links"><a href="#">Sponsor</a></li>
+        <li class="links"><a href="#">Join</a></li>
+        <li class="links"><a href="#">News</a></li>
+      </ul>
+    </header>
+    <h1>Hello sharing world!</h1>
+    <h2>CREATIVE SUMMER CONCERT 2023</h2>
+    <P class="primary-text">The Global Summit brings together the community of experts, academics, and activists who comprise the Creative Commons affiliate network in a different country every two years.
+      This year, we hope to expand our invitation list including organizations and individuals who want to work with us on shared projects that advance the cause of the Commons, free culture and open knowledge.
+      So if you're active and engaged in the worlds of open content and knowledge - free software advocates, wikipedians, Open knowledge, galleries, libraries, museums, archives, governments and foundations, lawyers, and activists - we hope you'll consider joining us this year to build a stronger, more vibrant commons together</P>
+      <p>Please contact us per Email for nay further questions about Summit concert show 2023!</p>
+      <strong id="subscribe">concert@hotmail.com</strong>
+  </section>
+  <section class="middle">
+    <h3>Summer concert series 2023 Logo</h3>
+    <p>The logo of summer concert 2023 was decided through the logo competition from 8, June to 7, July.
+      After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.
+      After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s concert as follows:
+    </p>
+      <div id="middle-logo">
+        <img src="./images/summer.png" alt="" class="summer-logo">
+      </div>
+  </section>
+  <section class="bottom">
+    <strong>See previous summer concert series</strong>
+    <p>Take a look at the past two Summer concert which took place in Buenos Aires and in Warsaw</p>
+    <div id="previous">
+    <div class="year1">
+      <h2>2022</h2>
+      <p>Summer Concert Series at Australia</p>
+    </div>
+    <div class="year2">
+      <h2>2021</h2>
+      <p>Summer Concert Series at Korea</p>
+    </div>
```

Comment:
- [ Required ] I think it would be better if your design for this section had the same background color as the one used in the Behance design for these two `divs`. Remember our goal is to match the design template provided as much as we can, please see the screenshot below and rectify this.

## Project requirements 
<img src="https://user-images.githubusercontent.com/16392046/225945846-69ebd9f6-134c-4eba-9c30-1b3bf20b5946.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/225945948-7bdba2be-3ae7-46c6-85fc-12b17051785d.png" width="375px" />